src/lib/TrackManager.ts

```diff
@@ -246,6 +248,16 @@ export async function loadTrackManager(url: string) {
             numberOfValuesPerPoint = 3;
         }
 
+        let datasetNdim = 3;
+        try {
+            const store = new HTTPStore(url);
+            const zattrsResponse = await store.getItem("points/.zattrs");
+            const zattrs = JSON.parse(new TextDecoder().decode(zattrsResponse));
+            datasetNdim = zattrs["ndim"];
```

Comment:
I have not used it extensively, but I believe you should be able to use `points.zattrs` like the following instead.

```typescript
const zattrs = await points.attrs.asObject();
datasetNdim = zattrs.ndim;
```

Definitely optional because you used the current approach previously. If you change one, you should probably change both.